Source/JavaScriptCore/wasm/WasmLLIntPlan.cpp

```diff
@@ -154,7 +154,7 @@ void LLIntPlan::didCompleteCompilation()
 
         m_entryThunks = FINALIZE_CODE(linkBuffer, JITCompilationPtrTag, "Wasm LLInt entry thunks");
         m_callees = m_calleesVector.data();
-        if (!m_moduleInformation->clobberingTailCalls().isEmpty())
+        if (m_moduleInformation->hasClobberingTailCalls())
```

Comment:
Now, we are always allocating this vector for all functions, so I think this `isEmpty()` check's meaning has been changed, right?